download.py

```diff
@@ -462,6 +462,15 @@ def append_asset(asset_data, **kwargs):  # downloaders=[], location=None,
             asset_thumbs_dir = paths.get_download_dirs("brush")[0]
             asset_thumb_path = os.path.join(asset_thumbs_dir, thumbnail_name)
             shutil.copy(thumbpath, asset_thumb_path)
+
+            # re-mark as asset in blender version >= 4.3
+            # but only if asset comes from a version older than that
+
+            if not utils.asset_from_newer_blender_version(
+                asset_data, blender_version=(4, 2, 9)
```

Comment:
Blender 4.2 is LTS and we can expect it to have more than 9 versions during the LTS period. Is there any particular reason for comparing to 4.2.9? I am affraid this is introducing a bug in the future when the code will be triggered for Blender 4.2.10, 4.2.11 etc.

Maybe it would be better to just convert the asset version to tupple and then compare the tuples in here. Using the quickly bended function feels hacky. If so, the function should be rewritten, so comparison to 4.3 tupple(4,3) is possible.